src/Graphormer.jl

```diff
@@ -0,0 +1,38 @@
+using Flux
+using Flux: @epochs, mse, params
+
+function Graphormer(input_dim, hidden_dim, output_dim, num_layers, num_heads)
```

Comment:
Please be consistent with the codebase. 
Layers should be structs, not functions, and should live in `src/layers/conv.jl`.

Reply:
Sure let me update it accordingly

Reply:
@CarloLucibello can you have a look at the updated script?